tests/user.test.js

```diff
@@ -129,4 +129,28 @@ describe('User', () => {
         });
     });
   });
+
+  describe('user functionality', () => {
```

Comment:
When rebasing or merging, remember to implement login(to get a token as we have got a working solution of getting a token). The test should work if you either make token verification optional or add a token.